c/prima.c

```diff
@@ -134,9 +162,24 @@ int prima_free_result(prima_result_t *result)
     return PRIMA_NULL_RESULT;
 }
 
+void prima_set_optional_values(prima_problem_t *problem, prima_options_t *options) {
+  // Look for any non-NAN values in the optional arguments and set the corresponding pointers.
+  if (isnan(problem->f0) == false) {
```

Comment:
For me, `if (! isnan(problem->f0))` is better than `if (isnan(problem->f0) == false)`.

Reply:
Sometimes I find the `!` is hard to see, do you mind if I do `if ( ! isnan(problem->f0))`?

Reply:
OK